### PR TITLE
Fix URL schemes in `dokku urls` output

### PR DESCRIPTION
**Pull request by lmars**
Previously, the presence of TLS certificates would cause `dokku urls` to output all https URLs, even for domains which were not configured to use TLS.

This change introduces a URLS file in the app's directory which contains the URLs with schemes based on whether they have been configured to use TLS in the nginx config.

### CODE REVIEW

File: plugins/nginx-vhosts/commands [hunk 1]
```diff
@@ -94,6 +95,11 @@ EOF
         echo "       Reloading nginx"
         restart_nginx
       fi
+
+      echo "-----> Generating app URLs..."
```
Comment (josegonzalez) [inline plugins/nginx-vhosts/commands]: Do we need the extra output? If so, you should use the helper in common for this :)

Comment (lmars) [inline plugins/nginx-vhosts/commands]: Not really needed no, dropped.

**Comment by michaelshobbs** [comment]
In reference to my comment on #1008, perhaps we can right a test that ensures we don't return `https` for domains that only partially match the cert CN. (i.e. CN=*.dokku.me and domain is www.test.dokku.me)

**Comment by lmars** [comment]
@michaelshobbs I have fixed the sub-subdomain match issue in #1008

**Pull request merged**